helios/models.py

```diff
@@ -244,6 +244,22 @@ def get_by_short_name(cls, short_name):
       return cls.objects.get(short_name=short_name)
     except cls.DoesNotExist:
       return None
+    
+  def save_questions_safely(self, questions):
+    """
+    Because Django doesn't let us override properties in a Pythonic way... doing the brute-force thing.
+    """
+    # verify all the answer_urls
+    for q in questions:
+      for answer_url in q['answer_urls']:
```

Comment:
no, this is just the way the data model is set up, `answer_urls` is the array of all URLs, one for each option for the question.